### PR TITLE
Add `campaigns` module to styleguide

### PR DESCRIPTION
**Pull request by stevoland**
Moves `ma-says` bubble out to a re-usable component and introduces an SVG sprite for small SVG icons.

### CODE REVIEW

File: app/assets/stylesheets/layouts/_campaigns.scss [hunk 1]
```diff
@@ -0,0 +1,63 @@
+.l-campaigns {
+    @include clearfix();
```
Comment (benbarnett) [inline app/assets/stylesheets/layouts/_campaigns.scss]: Too much indentation?

File: app/assets/stylesheets/layouts/_campaigns.scss [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  background: $white;
+  height: 0;
+  padding-bottom: 32.7288%;
+  position: relative;
+}
+
+.l-campaigns__hero img {
+  position: absolute;
+  display: block;
+  top: 0;
```
Comment (benbarnett) [inline app/assets/stylesheets/layouts/_campaigns.scss]: Is this necessary to specify?

File: app/assets/stylesheets/layouts/_campaigns.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  left: 0;
```
Comment (benbarnett) [inline app/assets/stylesheets/layouts/_campaigns.scss]: And this

File: app/views/shared/_svg_sprite.html.erb [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+  #
+  # <svg>
+  #   <use xlink:href="#svg-arrow-right"></use>
+  # </svg>
+%>
+<svg style="display: none;">
+  <symbol id="svg-arrow-right" viewBox="0 0 22 34">
+    <polygon points="0,5 5,0 22,17 5.1,34 0,29 12,17" />
+  </symbol>
+</svg>
```
Comment (benbarnett) [inline app/views/shared/_svg_sprite.html.erb]: Awesome :100:

Comment (benbarnett) [inline app/views/shared/_svg_sprite.html.erb]: Thoughts on moving this into an SVG file to assist caching?

File: app/views/styleguide/show.html.erb [hunk 5]
```diff
@@ -42,4 +42,9 @@
     <p>When the viewport exceeds 768px the buttons with fix to the right side of the screen</p>
     <%= render 'shared/sharing_buttons' %>
   </section>
+
```
Comment (benbarnett) [inline app/views/styleguide/show.html.erb]: Whitespace not needed

**Comment by darokel** [comment]
@stevoland @benbarnett @Guntrisoft Does this still need some work/updating or is it good to go in?

**Comment by benbarnett** [comment]
@darokel the comments above need addressing but @stevoland no longer on the project.

**Comment by Guntrisoft** [comment]
I'll work on those issues when I've finished up on the archive stuff

**Comment by darokel** [comment]
Sounds good @Guntrisoft

File: app/views/shared/_svg_sprite.html.erb (hunk 4 above)
Comment (stevoland) [inline app/views/shared/_svg_sprite.html.erb]: Unfortunately, no version of IE supports referencing external svgs with `use`. There is a polyfill: https://github.com/jonathantneal/svg4everybody

**Comment by benbarnett** [comment]
Merged here https://github.com/moneyadviceservice/publify/pull/75, so closing this.

**Pull request closed without merging**